source/js/buyers-guide/bg-main.js

```diff
@@ -76,8 +76,8 @@ let main = {
         document.body.classList.add(`react-loaded`);
         this.initPageSpecificScript();
         // bind custom analytics only once everything's up and loaded
-        AnalyticsEvents.init();
         PNIMobileNav.init();
+        AnalyticsEvents.init();
```

Comment:
If I understand correctly you are changing the order here because errors in `AnalyticsEvents.init()` prevent `PNIMobileNav.init()` from ever running?

Reply:
What do you think about adding a comment about the flakiness of the analytics initialization to avoid other scripts being added after it?

Reply:
Yeah I can do that!